embeddings/transformation/hf_transformation/sampling_transformation.py

```diff
@@ -81,3 +83,39 @@ def transform(self, data: datasets.DatasetDict) -> datasets.DatasetDict:
             return data
 
         return dataset
+
+
+class SampleSplitsHuggingFaceTransformation(SampleSplitTransformation):
+    def _train_test_split(
+        self, data: datasets.Dataset, test_fraction: float
+    ) -> datasets.DatasetDict:
+        return data.train_test_split(test_size=test_fraction, seed=self.seed)  # type: ignore
+
+
+class SampleSplitsStratifiedTransformation(SampleSplitTransformation):
+    def __init__(
+        self,
+        target_field_name: str,
+        dev_fraction: Optional[float] = None,
+        test_fraction: Optional[float] = None,
+        seed: int = 441,
+    ):
+        self.target_field_name = target_field_name
+        super().__init__(dev_fraction, test_fraction, seed)
```

Comment:
```suggestion
        super().__init__(dev_fraction, test_fraction, seed)
        self.target_field_name = target_field_name
```

calling parent's init at the beginning  allows usage of attributes, properties etc, which would be helpful in case of refactor